### PR TITLE
added bounties

### PR DESCRIPTION
**Pull request by bwhm**
No longer WIP!

### CODE REVIEW

**Review by DzhideX — APPROVED** [review]
Looks good!

- As said though, any tags that we didn't specifically add filters won't be "filterable"!

File: bounties-overview/bounties-status.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      "links": ["https://github.com/Joystream/community-repo/issues/87"],
+      "reward": 300,
+      "tags": ["Newcomers","Research"],
+      "format": "Free for All",
+      "status": "Announced"
+    },
+    {
+      "id": 10,
+      "title": "Upload Public Domain Content",
+      "description": "Upload videos to our testnet, and earn up to $5 per entry! We believe there are some hidden gems available out there, that is licensed permissive.",
```
Comment (blrhc) [inline bounties-overview/bounties-status.json:40]:
```suggestion
      "description": "Upload videos to our testnet, and earn up to $5 per entry! We believe there are some hidden gems available out there, that are licensed permissively.",
```

File: bounties-overview/bounties-status.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      "links": ["https://github.com/Joystream/community-repo/issues/86"],
+      "reward": 300,
+      "tags": ["Research"],
+      "format": "Closed",
+      "status": "Announced"
+    },
+    {
+      "id": 9,
+      "title": "Repo/Docs Improvements",
+      "description": "Improve the quality of our documentation on Github, by making PRs and ",
```
Comment (blrhc) [inline bounties-overview/bounties-status.json:29]:
```suggestion
      "description": "Improve the quality of our documentation on GitHub, by making PRs and issues.",
```

File: bounties-overview/bounties-status.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      "links": ["https://github.com/Joystream/community-repo/issues/85"],
+      "reward": 300,
+      "tags": ["Coding","Research"],
+      "format": "Closed",
+      "status": "Announced"
+    },
+    {
+      "id": 8,
+      "title": "Ledger on Joystream",
+      "description": "Research how well the Ledger hardware wallet is supported, how to use them on the network, derivation paths and key recovery.",
```
Comment (blrhc) [inline bounties-overview/bounties-status.json:18]:
```suggestion
      "description": "Research how well the Ledger hardware wallets are supported, how to use them on the network, derivation paths and key recovery.",
```

File: bounties-overview/README.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+|7  |Joystream Player Loading     |#85   |15.02.21 |$400  |[n](https://testnet.joystream.org/#/forum/threads/n)      |Announced       |NA   |NA                                                  |
+|8  |Ledger on Joystream          |#86   |15.02.21 |$300  |[n](https://testnet.joystream.org/#/forum/threads/n)      |Announced       |NA   |NA                                                  |
+|9  |Repo/Docs Improvements       |#87|15.02.21 |$400  |[n](https://testnet.joystream.org/#/forum/threads/n)      |Announced       |NA   |NA                                                  |
+|10 |Upload Public Domain Content |#88   |15.02.21 |$5*  |[n](https://testnet.joystream.org/#/forum/threads/n)      |Announced       |NA   |NA                                                  |
+|11 |Design Community Repo Banner |#89   |15.02.21 |$250  |[n](https://testnet.joystream.org/#/forum/threads/n)      |Announced       |NA   |NA                                                  |
+
+## Bounties Management
+Part of the job for the Council is to manage these bounties. The tasks associated with that is:
```
Comment (blrhc) [inline bounties-overview/README.md:16]:
```suggestion
Part of the job for the Council is to manage these bounties. The tasks associated with that are:
```

File: bounties-overview/README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+1. Familiarize themselves with the tasks specified in the issue
```
Comment (blrhc) [inline bounties-overview/README.md:17]:
```suggestion
1. Familiarizing themselves with the tasks specified in the issue
```

File: bounties-overview/README.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+2. Seek further information from Jsgenesis on any ambiguous or missing items
```
Comment (blrhc) [inline bounties-overview/README.md:18]:
```suggestion
2. Seeking further information from Jsgenesis on any ambiguous or missing items
```

File: bounties-overview/README.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+3. Decide on the format, full workflow and process
```
Comment (blrhc) [inline bounties-overview/README.md:19]:
```suggestion
3. Deciding on the format, full workflow and process required
```

File: bounties-overview/README.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+4. Create a forum thread with:
```
Comment (blrhc) [inline bounties-overview/README.md:20]:
```suggestion
4. Creating a forum thread with:
```

File: bounties-overview/README.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+  - Link to the issue with full description
+  - Specify the format and workflow
+  - If applicable
```
Comment (blrhc) [inline bounties-overview/README.md:23]:
```suggestion
  - If applicable:
```

File: bounties-overview/README.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+    - list what it takes to be assigned the bounty
+    - assign an admin
```
Comment (blrhc) [inline bounties-overview/README.md:25]:
```suggestion
    - assign a dedicated manager for the bounty
```

File: bounties-overview/README.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+    - set milestones/timelines
+5. Update the [json](/bounties-overview/bounties-status.json) in accordance with the [schema](/bounties-overview/bounties). See explanation [here](#bounties-schema).
+6. Assign a community member`*`, and update json
+7. Follow up as required
+8. Once a non-final spending proposal`**` is made:
+  - Ensure it's in line with the workflow set
+  - Review the work submitted
+  - Approve if appropriate
+9. Once the final spending proposal is made:
+  - Ensure it's in line with the workflow set
```
Comment (blrhc) [inline bounties-overview/README.md:35]:
```suggestion
  - Ensure it's in line with the workflow set out
```

**Review by blrhc — APPROVED** [review]
Approved, but with a couple of structural suggestions.

**Comment by bwhm** [comment]
Implemented them all!
Merge if ok :)

**Pull request merged**